app/src/main/java/org/mozilla/tv/firefox/ScreenController.kt

```diff
@@ -61,9 +61,9 @@ class ScreenController {
                 .commit()
     }
 
-    fun showBrowserScreenForCurrentSession(fragmentManager: FragmentManager, session: Session) {
+    fun showBrowserScreenForCurrentSession(fragmentManager: FragmentManager, session: Session, menu: Boolean) {
         val fragment = fragmentManager.findFragmentByTag(WebRenderFragment.FRAGMENT_TAG) as WebRenderFragment?
-        if (fragment != null && fragment.session == session) {
+        if (fragment != null && fragment.session == session && !menu) {
```

Comment:
I'm not clear on what `menu` means here.  Could we rename it?

Is it 'overlay is currently open'?  If so, do we still need the first two parts of this conditional statement or is `menu` enough?

Reply:
This is needed for closing overlay. Originally the conditional statement was there to check if BrowserFragment is already displaying something in case of `MainActivity.OnCreate()` gets called - caching browsing state perhaps?